post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.adoc

```diff
@@ -12,21 +12,20 @@ The Multiarch Tuning Operator is a Technology Preview feature only. Technology P
 For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
 ====
 
-The Multiarch Tuning Operator enhances the operational experience within multi-architecture clusters, and single-architecture clusters that are migrating to a multi-architecture compute configuration. 
+The Multiarch Tuning Operator enhances the operational experience within multi-architecture clusters, and single-architecture clusters that are migrating to a multi-architecture compute configuration.
 
-This Operator implements the `ClusterPodPlacementConfig` object to support architecture-aware workload scheduling.
+This Operator implements the `clusterpodplacementconfigs` custom resource (CR) to support architecture-aware workload scheduling.
 
-When you create the `ClusterPodPlacementConfig` object, the Multiarch Tuning Operator deploys an operand. This operand automates the configuration of scheduling predicates for the workloads based on their supported architectures. It uses the `kubernetes.io/arch` label set by the kubelet in the node objects.
+To enable architecture-aware workload scheduling, you must create the `ClusterPodPlacementConfig` object. When you create the `ClusterPodPlacementConfig` object, this Operator deploys an operand.
 
-When a pod is created, the operand configures the `nodeAffinity` field in the pod specification by computing the subset of architectures that the pod can support.
+When a pod is created, the operand performs the following actions: 
 
-Before a pod enters the scheduling cycle, the operand adds the `multiarch.openshift.io/scheduling-gate` scheduling gate that prevents the scheduling of the pod until the operand completes the following actions:
+. Add the `multiarch.openshift.io/scheduling-gate` scheduling gate that prevents the scheduling of the pod.
+. Compute a scheduling predicate that includes the supported architecture values for the `kubernetes.io/arch` label. 
```

Comment:
whether changing to below more clear? @aleskandro please correct me if I am wrong.
```suggestion
. Compute a scheduling predicate that includes the supported architecture values for the node `kubernetes.io/arch` label matching. 
```

Reply:
@aleskandro 
WDYT?

Reply:
nope, it doesn't. We patch the pods, not the nodes.

Reply:
thanks.